app/views/node_info/_statistics.haml

```diff
@@ -0,0 +1,21 @@
+-#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
```

Comment:
I feel bad about including this header always.

What about just;

> # This file is licensed under the Affero General Public License version 3 or later.  See
> # the COPYRIGHT file.

Diaspora Inc doesn't exist any more and this file wasn't amde in 2010-2011 and the current files are not even copyright Diaspora, Inc according to the agreement with the FSSN.

I know this is not related to this PR, but still for this file the first sentence could be removed ;) Also, do files even need the license? Isn't a repo license enough?